handlers/label_handler.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from handlers.EventHandler import EventHandler
+
+class LabelHandler(EventHandler):
+    async def handle_event(self, data, postgres_client):
+        # Implement your logic for handling issue events here
+        try:
+            if data.get("action") == 'edited' and 'name' in data.get("changes"):
+                label_name = data["label"]["name"].lower()
+                if 'c4gt' in label_name and label_name not in ['c4gt community', 'dmp 2024']:
```

Comment:
This seems to be repeating. Saw in another class as well. Can we keep it in 1 place?